src/App/constants.ts

```diff
@@ -23,7 +23,7 @@ const HEADER_LINKS: Link[] = [
   },
   {
     title: "Giving Partners (CSR)",
-    href: `${BASE_URL}/giving-partners-csr/`,
+    href: `/wp/giving-partners-csr/`,
```

Comment:
@SovereignAndrey, unlike posts, which follows a certain layout (image | header | content) and thus can be styled in a generic way, pages have no similarities at all, 

CSR
![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/0d2a99b8-d57f-4f99-83e4-496c386e8785)

ABOUT
![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/00c6dc54-a0d7-492f-91a9-6b084b233fdf)

and to render them in a generic page `wp/:slug` and expect them look like what's currently in marketing site is something I initially think I couldn't pull of - but I am still researching on this 🙏

Currently trying to get whatever styling tooling used in the rendered html, and apply it according ly
```html
\t\t<div data-elementor-type=\"wp-page\" data-elementor-id=\"6142\" class=\"elementor elementor-6142\" data-elementor-post-type=\"page\">\n\t\t\t\t\t\t\t\t\t<section class=\"elementor-section elementor-top-section elementor-element elementor-element-4221330 elementor-section-full_width elementor-{&quot;background_background&quot;:&quot;classic&quot;}\">\n\t\t\t\t\t\t<div class=\"elementor-container elementor-column-gap-default\">\n\t\t\t\t\t<div class=\"elementor-column elementor-col-100 elementor-top-column elementor-element elementor-element-2846b28\" data-id=\"2846b28\" data-element_type=\"column\">\n\t\t\t<div class=\"elementor-widget-wrap\">\n\t\t\t\t\t\t\t\t\t<\/div>\n\t\t<\/div>\n\t\t\t\t\t\t\t<\/div>\n\t\t<\/section>\n\t\t\t\t<section class=\"elementor-section elementor-top-section elementor-element elementor-element-fa27b28 elementor-section-boxed elementor-section-height-default elementor-section-height-default\" data-id=\"fa27b28\" data-element_type=\"section\">\n\t\t\t\t\t\t<div class=\"elementor-container elementor-column-gap-default\">\n\t\t\t\t\t<div class=\"elementor-column elementor-col-100 elementor-top-column elementor-element elementor-element-d2676d3\" data-id=\"d2676d3\" data-element_type=\"column\">\n\t\t\t<div class=\"elementor-widget-wrap elementor-element-populated\">\n\t\t\t\t\t\t\t\t<div class=\"elementor-element elementor-element-d3a4670 elementor-widget elementor-widget-premium-addon-title\" data-id=\"d3a4670\" data-element_type=\"widget\" data-widget_type=\"premium-addon-title.default\">\n\t\t\t\t<div class=\"elementor-widget-container\">\n\t\t\t\n\t<div class=\"premium-title-container style7\">\n\t\t<h1 class=\"premium-title-header premium-title-style7\">\n\t\t\t\t\t\t\t\t\t\t\t\t<span class=\"premium-title-style7-stripe-wrap\">\n\t\t\t\t\t\t<span class=\"premium-title-style7-stripe\"><\/span>\n\t\t\t\t\t<\/span>\n\t\t\t\t\t\t\t\t<div class=\"premium-title-style7-inner\">\n\t\t\t\n\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t<span class=\"premium-title-text\" >\n\t\t\t\tPrivacy Policy\t\t\t<\/span>\n\t\t\t\t\n\t\t\t\t\t\t\t<\/div>\n\t\t\t\t\t\t\t\t<\/h1>\n\t<\/div>\n\n\t\t\t\t<\/div>\n\t\t\t\t<\/div>\n\t\t\t\t<div class=\"elementor-element elementor-element-131edbf elementor-widget elementor-widget-text-editor\" data-id=\"131edbf\" data-element_type=\"widget\" data-widget_type=\"text-editor.default\">\n\t\t\t\t<div class=\"elementor-widget-container\">\n\t\t\t<style>\/*! elementor - v3.17.0 - 08-11-2023 *\/\n.elementor-widget-text-editor.elementor-drop-cap-view-stacked .elementor-drop-cap{background-color:#69727d;color:#fff}.elementor-widget-text-editor.elementor-drop-cap-view-framed .elementor-drop-cap{color:#69727d;border:3px solid;background-color:transparent}.elementor-widget-text-editor:not(.elementor-drop-cap-view-default) .elementor-drop-cap{margin-top:8px}.elementor-widget-text-editor:not(.elementor-drop-cap
```

Reply:
> UPDATE: looking into `page.template` attribute that could be used to get template / theme

@SovereignAndrey, found that these resources, `theme`,`template` needs additional authentication for currently returns `401`. Is there a way I could access the wordpress content directly?